subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -596,19 +552,161 @@ static void win_offset_calc(struct ll_conn *conn_curr, uint8_t is_select,
 	*offset_max = offset_index;
 }
 #endif /* CONFIG_BT_LL_SW_LLCP_LEGACY */
-
 #endif /* CONFIG_BT_CTLR_CONN_PARAM_REQ */
+#endif /* CONFIG_BT_CONN */
+
+static uint8_t after_xxxx_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
+				   ticker_op_match_func ticker_match_op_cb,
+				   ull_hdr_get_func ull_hdr_get_cb,
+				   uint32_t *ticks_anchor,
+				   uint32_t *ticks_to_expire_xxxx,
+				   uint32_t *ticks_slot_xxxx)
+{
+	uint32_t ticks_to_expire_prev;
+	uint32_t ticks_slot_abs_prev;
+	uint32_t ticks_to_expire;
+	uint8_t ticker_id_prev;
+	uint8_t ticker_id;
+
+	ticks_slot_abs += HAL_TICKER_US_TO_TICKS(EVENT_JITTER_US << 3);
+
+	ticker_id = ticker_id_prev = 0xff;
```

Comment:
Since you are adding the `U` suffix on the others, might as well add that here too: 

```suggestion
	ticker_id = ticker_id_prev = 0xffU;
```

Reply:
replaced with `TICKER_NULL`